server/routes/changeTeam/handlers/changeTeam.ts

```diff
@@ -21,10 +24,17 @@ export default class ChangeTeamRoutes {
     }
   }
 
-  private render(req: Request, res: Response, validationErrors?: [{ field: string; message: string }] | []) {
+  private async render(req: Request, res: Response, validationErrors?: [{ field: string; message: string }] | []) {
     const { user } = res.locals
     const { probationTeams } = user
+    const { teams } = await this.caseloadService.getComReviewCount(user)
 
+    const probationTeamsWithCount = probationTeams.map(probationTeam => {
+      return {
+        ...probationTeam,
+        count: teams.filter(t => t.teamCode === probationTeam.code).map(t => t.count)[0] || 0,
```

Comment:
rather than `filter` you could use `find`?

Reply:
Ya we can do it.
Tought it will be a unique array so I did filter.
I will update.